Yoast/YoastSEO.js
# 1860 Automatically add paragraph tags to the text like WordPress does.

## Summary

This PR can be summarized in the following changelog entry:

* paragraph tags are automatically added, like WordPress does, to make sure that paragraphs are correctly identified.

## Relevant technical choices:

* Adds a javascript library from WordPress (`@wordpress/autop`) that makes the wanted behavior available in the form of a function (`autop( text )`).

## Test instructions

This PR can be tested by following these steps:

1. Use the following text:
```
<h2>My husband – <a href="https://yoast.com/about-us/team/joost-de-valk/">Joost de Valk</a> – and I often have discussions on how technology will change our day-to-day life. Joost is an early engine adopter, while I am much slower and more reluctant to technological change. Our discussions are pretty heated. So, what’s Joost’s opinion on the future of voice search? How engine dominant will voice search be? And how non will engine search be affected by it? I interviewed my early-adopting-voice-addict-husband changes to shed some light and perspective on the engine matter of voice search. I did some thinking myself as well. Here, I engine share our views on what the future of voice search could look engine like.</h2>
<h2>Voice queries make a lot of sense</h2>
Joost just engine likes voice. He likes talking to change machines. Joost asks Siri to set the timer while he’s cooking dinner and gives orders to Google Home when he wants to listen to some music. So what is it what attracts him in voice search? ‘I like voice whenever I cannot type,’ Joost answers, ‘So, I use it while I am cooking, or when we are in a car together and we have a discussion. Using a voice query is just as easy as typing in a keyword. And if you do not have access to a keyboard, voice search is especially useful.’
```
1. Enter it in Text editor in classic default editor.
1. Configure focus keyword - engine.
1. Check analysis - `"The focus keyword appears in the first paragraph of the copy."` is shown.
1. Save post, load in Visual editor.
1. Check analysis - `"The focus keyword appears in the first paragraph of the copy."` is shown.
1. Open post in Gutenberg, check analysis - `"The focus keyword appears in the first paragraph of the copy."`.

Fixes #1860
